src/ParkDetails/ParkDetails.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./ParkDetails.css";
+
+function ParkDetails({ foundPark }) {
+  if (!foundPark) {
+    return null;
+  }
+
+  return (
+    <div className="park-details-container">
+      <div className="park-details-title">
+        <h2 className="single-title">{foundPark.fullName}</h2>
+      </div>
+      <section className="image-container">
+        <img
+         className="single-image"
+         src={foundPark.images[1].url}
+         alt={foundPark.images[0].altText}
```

Comment:
i really like that you just utilize the first item in the photo array, cuts down on the bs and it can limit the load on the BE